lib/functions/plugins.bash

```diff
@@ -42,7 +42,7 @@ plugin_list_command() {
     ) | awk '{ if (NF > 1) { printf("%-28s", $1) ; $1="" }; print $0}'
   else
     display_error 'No plugins installed'
```

Comment:
```suggestion
    printf "No plugins installed\n"
```
since it's no longer an error, i think it's better to use stdout instead of stderr

Reply:
In the future it may be worth changing this depending on if standard output is a tty. But I don' think it's worth changing that right now